### DISCUSSION TITLE
Won't allow me to approve changes

### DISCUSSION THREAD

**Discussion by mbrasseau** [community/community#21667] [Other Feature Feedback, Questions, & Ideas]
<p>I just completed the tutorial about Pages.  Afterward, I made a change to my page.  Then I commited the change and requested a pull request.  It now states that it needs to be reviewed.  When I attempt to review and approve, it does not allow me to approve my own page, it only allows me to comment.</p>
<p>How do I approve my own page.  I’m the only one on this account.</p>

**Comment by mbrasseau** [comment, marked as answer]
<p>I have figured it out. I had to make a change to</p>
<p>Settings --&gt; Branches </p>
<p>and deselect the option for Required Reviews.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/mbrasseau">@mbrasseau</a>, thanks a lot for coming back and sharing what worked for you! We’re looking forward to seeing you around!</p>
